src/cgl.rs

```diff
@@ -103,6 +137,7 @@ pub const CORE_INTEGER_ATTRIBUTES: &'static [CGLPixelFormatAttribute] =
       kCGLPFAOpenGLProfile,
       kCGLPFAVirtualScreenCount];
 
+#[link(name = "OpenGL", kind = "framework")]
```

Comment:
Yeah, this looks right to me. Linking to the framework multiple times shouldn't hurt anything.